src/components/Spawn/spawnHostModal/getFormSchema.tsx

```diff
@@ -421,7 +420,6 @@ export const getFormSchema = ({
         "ui:data-cy": "distro-input",
```

Comment:
I'm no longer able to select any workstation distro in the `SpawnHostModal`. 🤔 The page just crashes